src/Profile_User.php

```diff
@@ -771,7 +771,7 @@ public static function getUserProfiles($user_ID, $sqlfilter = [])
         $profiles = [];
 
         $where = ['users_id' => $user_ID];
-        if (count($sqlfilter) > 0) {
+        if (is_array($sqlfilter) && count($sqlfilter) > 0) {
```

Comment:
This will indeed fix the issue, but is it expected that `$sqlfilter` is something else than an array? A bas usage of this parameter eleswhere may be the issue.

Reply:
I did not find in code where we can have something else than an array.

Reply:
_2nd client with the same bug_
I don't understand, because there the function is called without the `$sqlfilter` argument, so it is an empty array.